### PR TITLE
Show Offer name on Admin Page Issue

### PR DESCRIPTION
**Pull request by ms-mgrant**
Fixes issue #218 

Copy of PR #270 by @k-granados. This one resolves the merge conflicts.

### CODE REVIEW

File: deployment/Database/AMP-DB-5.1.sql [hunk 1]
```diff
@@ -481,6 +482,8 @@ PRIMARY KEY CLUSTERED
 GO
 ALTER TABLE [dbo].[Subscriptions] ADD  CONSTRAINT [DF_Subscriptions_AMPSubscriptionId]  DEFAULT (newid()) FOR [AMPSubscriptionId]
 GO
+ALTER TABLE [dbo].[Subscriptions] ADD PlanGUId uniqueidentifier NULL
```
Comment (santhoshb-msft) [inline deployment/Database/AMP-DB-5.1.sql:485]: Why do we have the PlanId twice?

File: src/SaaS.SDK.PublisherSolution/Controllers/HomeController.cs [hunk 2]
```diff
@@ -246,7 +246,16 @@ public IActionResult Subscriptions()
                     foreach (var subscription in allSubscriptionDetails)
                     {
                         SubscriptionResultExtension subscriptionDetailExtension = this.subscriptionService.PrepareSubscriptionResponse(subscription);
-                        Plans planDetail = this.planRepository.GetById(subscriptionDetailExtension.PlanId);
+                        Plans planDetail = new Plans();
+                        if (subscriptionDetailExtension.PlanGUId != null)
```
Comment (santhoshb-msft) [inline src/SaaS.SDK.PublisherSolution/Controllers/HomeController.cs:250]: please add a service function to get the offername by planIdGuid.

File: deployment/Database/AMP-DB-5.1.sql (hunk 1 above)
Comment (ms-mgrant) [reply deployment/Database/AMP-DB-5.1.sql:485]: This statement adds the PlanGUId column if the subscriptions table already exists. CREATE TABLE fails if the subscriptions table already exists for returning users, so the new column is never added.

**Comment by ms-mgrant** [comment]
Adds PlanGuid column:
image

**Comment by santhoshb-msft** [comment]
@ms-mgrant @k-granados  Bacpac file seems to be corrupt.

**Comment by ms-mgrant** [comment]
@santhoshb-msft  Resolved corrupt bacpac file

**Pull request closed without merging**